salt/master_utils/install_tracking_s3_monitor.sls

```diff
@@ -0,0 +1,24 @@
+
+install_monitor_script:
+  file.managed:
+    - name: /usr/local/sbin/check-s3-tracking-logs.sh
+    - contents: |
+        #!/usr/bin/env bash
+        BUCKETS="odl-residential-tracking-data"
+        # -e means fail the script on error ...
+        set -e
+        today=`date +"%F"`
+        for bucket in $BUCKETS; do
+            # ... and this will return error status for no matching objects:
+            aws s3 ls s3://$bucket/logs/$today
+        done
+        curl -s '{{ __vault__::secret-operations/global/healthchecks/mitx-tracking-s3>data>value }}'
```

Comment:
This needs to be passed as pillar data. The vault pillar renderer doesn't work in state files.

Reply:
Would the `master.config` pillar be the best place? Or one of the neighboring ones under `master`?

Reply:
I've made an attempt to fix it in commit 98a25d34
Can change it, if necessary!